src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_SwitchExpression.cs

```diff
@@ -92,10 +92,7 @@ private BoundExpression LowerSwitchExpression(BoundConvertedSwitchExpression nod
                     sectionBuilder.Add(_factory.Label(arm.Label));
                     var loweredValue = _localRewriter.VisitExpression(arm.Value);
                     if (GenerateSequencePoints)
-                    {
```

Comment:
>{ [](start = 20, length = 1)

If we are going to keep the ```if``` at the end, consider keeping braces as well.

Reply:
All single-line if statements in this source file are without braces. I prefer to maintain that consistency.

---
In reply to: [410986252](https://github.com/dotnet/roslyn/pull/43430#discussion_r410986252) [](ancestors = 410986252)